docs/configuration.md

```diff
@@ -2955,6 +2955,12 @@ Spark uses [log4j](http://logging.apache.org/log4j/) for logging. You can config
 `log4j.properties` file in the `conf` directory. One way to start is to copy the existing
 `log4j.properties.template` located there.
 
+By default, Spark adds 1 record to the MDC (Mapped Diagnostic Context): `taskName`, which shows something
+like `task 1.0 in stage 0.0`. You can add `%X{taskName}` to your patternLayout in
```

Comment:
`patternLayout ` -> `PatternLayout `. 

Could you give an example to show how to use it in this document? For example, use an example to show how to specify your application names/identifiers.

Reply:
@gatorsmile were you able to figure this out? My MDC values are not propagating in my logs after following this same procedure.

Reply:
@alefischer13 Just a guess, but it looks like this was changed in 54e702c so that the MDC key still includes the `mdc.` prefix.